cli/app/ui_worktrees.go

```
@@ -0,0 +1,1482 @@
+	}
+}
+
+func renderWorktreeBadges(item serverapi.WorktreeView, selected bool, theme string) []string {
+	p := uiPalette(theme)
+	badges := make([]string, 0, 4)
+	base := lipgloss.NewStyle()
+	if selected {
+		base = base.Background(p.modeBg)
+	}
+	badge := func(text string, fg lipgloss.TerminalColor) string {
+		return base.Copy().Foreground(fg).Bold(true).Render("[" + text + "]")
```

golangci-lint (2.11.4)

[error] SA1019: base.Copy is deprecated: to copy just use assignment (i.e. a := b). All methods also return a new style. 

(staticcheck)